### PR TITLE
APIS-5296 - Remove runMode/env from configuration

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala [hunk 1]
```diff
@@ -68,7 +68,7 @@ class UpliftVerificationExpiryJobConfigProvider @Inject()(val runModeConfigurati
   with Provider[UpliftVerificationExpiryJobConfig] {
 
   override def get() = {
-    val jobConfig = runModeConfiguration.underlying.as[Option[JobConfig]](s"${runMode.env}.upliftVerificationExpiryJob")
+    val jobConfig = runModeConfiguration.underlying.as[Option[JobConfig]](s"upliftVerificationExpiryJob")
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala:71]: No need for string interpolation here

File: app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala [hunk 2]
```diff
@@ -85,7 +85,7 @@ class MetricsJobConfigProvider @Inject()(val runModeConfiguration: Configuration
   with Provider[MetricsJobConfig] {
 
   override def get() = {
-    val jobConfig = runModeConfiguration.underlying.as[Option[JobConfig]](s"${runMode.env}.metricsJob")
+    val jobConfig = runModeConfiguration.underlying.as[Option[JobConfig]](s"metricsJob")
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala:88]: No need for string interpolation here

File: app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala [hunk 3]
```diff
@@ -107,7 +107,7 @@ class ApiStorageConfigProvider @Inject()(val runModeConfiguration: Configuration
   extends Provider[ApiStorageConfig] {
 
   override def get() = {
-    val disableAwsCalls = runModeConfiguration.getOptional[Boolean](s"${runMode.env}.disableAwsCalls").getOrElse(false)
+    val disableAwsCalls = runModeConfiguration.getOptional[Boolean](s"disableAwsCalls").getOrElse(false)
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala:110]: No need for string interpolation here

File: app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala [hunk 4]
```diff
@@ -137,7 +137,7 @@ class EmailConfigProvider @Inject()(val runModeConfiguration: Configuration, run
 
   override def get() = {
     val url = baseUrl("email")
-    val devHubBaseUrl = ConfigHelper.getConfig(s"${runMode.env}.devHubBaseUrl", runModeConfiguration.getOptional[String](_))
+    val devHubBaseUrl = ConfigHelper.getConfig(s"devHubBaseUrl", runModeConfiguration.getOptional[String](_))
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala:140]: No need for string interpolation here

**Review by peteslater-ee — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala (hunk 1 above)
Comment (AndySpaven) [reply app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala:71]: Drat

**Review by peteslater-ee — APPROVED** [review]

**Pull request merged**